### PR TITLE
Auto test smoke tests only

### PR DESCRIPTION
**Pull request by dangood84**
- Added cut-down version of Create project journey in "Test-env-smoke-tests" folder
- Abandoned using @cypress/grep as it will only filter if all tests are tagged with something (e.g. if we don't want to tag dev tests as @dev it'll run them anyway even if we're just looking for @smoke)
- Added a cy:smoke command in package.json to just run the tests in the "Test-env-smoke-tests" folder

### CODE REVIEW

**Comment by danielryannimble** [comment]
can we hold of merging these until the yml stuff is sorted

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/support/e2e.ts [hunk 1]
```diff
@@ -17,8 +17,6 @@
 import { AuthenticationInterceptorParams } from "cypress/auth/authenticationInterceptor";
 import "./commands";
 import { RuleObject } from "axe-core";
-const registerCypressGrep = require('@cypress/grep')
-registerCypressGrep()
```
Comment (danielryannimble) [inline Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/support/e2e.ts:21]: we need this left in

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress.config.ts [hunk 2]
```diff
@@ -41,7 +41,6 @@ export default defineConfig({
                     return null;
                 },
             });
-            require('@cypress/grep/src/plugin')(config);
```
Comment (danielryannimble) [inline Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress.config.ts:44]: we need this left in

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+import summaryPage from "cypress/pages/task-summary-base";
+
+    describe("Smoke Test Create an individual project - (Up Until Check your answers Page)", { tags: ['@smoke'] }, () => {
+        beforeEach(() => {
+            cy.login({ role: ProjectRecordCreator });
+            cy.visit('/');
+        });
+
+        it("Should Smoke Test go through create a project wizard happy path up until Check your answers page", { tags: ['@smoke'] }, () => {
+            const temporaryProjectId = dataGenerator.generateTemporaryId();
```
Comment (danielryannimble) [inline Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts:15]: just needs tagging as 'smoke'

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts [hunk 4]
```diff
@@ -0,0 +1,177 @@
+import { Logger } from "cypress/common/logger";
+import { ProjectRecordCreator } from "cypress/constants/cypressConstants";
+import dataGenerator from "cypress/fixtures/dataGenerator";
+import createProjectPage from "cypress/pages/createProject/createProjectPage";
+import homePage from "cypress/pages/homePage";
+import summaryPage from "cypress/pages/task-summary-base";
+
+    describe("Smoke Test Create an individual project - (Up Until Check your answers Page)", { tags: ['@smoke'] }, () => {
```
Comment (danielryannimble) [inline Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts:8]: i don't think we need the tag here

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+import homePage from "cypress/pages/homePage";
+import summaryPage from "cypress/pages/task-summary-base";
+
+    describe("Smoke Test Create an individual project - (Up Until Check your answers Page)", () => {
+        beforeEach(() => {
+            cy.login({ role: ProjectRecordCreator });
+            cy.visit('/');
+        });
+
+        it("Should Smoke Test go through create a project wizard happy path up until Check your answers page", () => {
```
[the remark this answers is not in the document]
Comment (dangood84) [reply Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts]: Fixed in latest commit.  Will repush before dev standup 👍

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/support/e2e.ts (hunk 1 above)
Comment (dangood84) [reply Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/support/e2e.ts:21]: Done!

File: Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts (hunk 4 above)
Comment (dangood84) [reply Dfe.ManageFreeSchoolProjects/Dfe.ManageFreeSchoolProjects.CypressTests/cypress/e2e/Test-env-smoke-tests/smoke-create-individual-project-to-check-answers-page.cy.ts:8]: All done Thanks!

Repushing this up to my remote branch 👍

**Pull request closed without merging**